### ISSUE TITLE
Trimethylguanosine synthase, question for Rhea

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5179]
as far as I can see this enzyme acts on snRNA and snoRNA

Using Human as an example:
https://www.uniprot.org/uniprotkb/Q96RS0/entry
The description appears to confirm this:

Catalyzes the 2 serial methylation steps for the conversion of the 7-monomethylguanosine (m7G) caps of snRNAs and snoRNAs to a 2,2,7-trimethylguanosine (m(2,2,7)G) cap structure. The enzyme is specific for guanine, and N7 methylation must precede N2 methylation. Hypermethylation of the m7G cap of U snRNAs leads to their concentration in nuclear foci, their colocalization with coilin and the formation of canonical Cajal bodies (CBs). Plays a role in transcriptional regulation

but the Rhea catalytic activity refers to an mRNA
a 5'-end (N7-methyl 5'-triphosphoguanosine)-ribonucleoside in mRNA + S-adenosyl-L-methionine = a 5'-end (N2,N7-dimethyl 5'-triphosphoguanosine)-ribonucleoside in mRNA + H+ + S-adenosyl-L-homocysteine

I guess my question is do we have any examples of mRNA TGM caps? I did not come across this activity for mRNA (although it does have different modifications to the 	
7-methylguanosine cap)

The reference used for Rhea
https://pubmed.ncbi.nlm.nih.gov/18775984/
does not refer to mRNA (at least in the abstract, it's (TMG) caps characteristic of spliceosomal small nuclear RNAs

thoughts?
@kaxelsen 
@sjm41

**Comment by kaxelsen** [comment]
The human entry was recently updated with two new reactions working in snRNA and snoRNA, respectively. These reactions replaced RHEA:67620 (mRNA) with RHEA:78471 and RHEA:78475. RHEA:67620 was made obsolete.
RHEA:78471 a 5'-end (N7-methyl 5'-triphosphoguanosine)-ribonucleoside in snRNA + S-adenosyl-L-methionine = a 5'-end (N2,N7-dimethyl 5'-triphosphoguanosine)-ribonucleoside in snRNA + H+ + S-adenosyl-L-homocysteine  
RHEA:78475 a 5'-end (N7-methyl 5'-triphosphoguanosine)-ribonucleoside in snoRNA + S-adenosyl-L-methionine = a 5'-end (N2,N7-dimethyl 5'-triphosphoguanosine)-ribonucleoside in snoRNA + H+ + S-adenosyl-L-homocysteine 

The updates will become public in the next release, 2024_03 scheduled for May 29

So, the answer to your question is: No, there are no TGM caps on mRNA, and we have already made the necessary changes in UniProtKB.

**Comment by ValWood** [comment]
Fantastic , thanks!

**Comment by ValWood** [comment]
Another question.

Does RHEA have a term that is agnostic for the RNA (I may keep the GO term and create descendants for the sn/sno versions.
Note that this activity also operates on telomerase, and based on a comment I added recently but forgot about , on
nematode mRNAs that undergo trans-splicing of a 5'-capped leader sequence.

**Comment by kaxelsen** [comment]
No, we do not have a parent reaction for the two reactions with snRNA and snoRNA.
